src/main/java/mServer/crawler/sender/wdr/WdrDeserializerBase.java

```diff
@@ -0,0 +1,24 @@
+package mServer.crawler.sender.wdr;
+
+public abstract class WdrDeserializerBase extends HtmlDeserializerBase {
+    
+    private static final String URL_ROOT = "http://www1.wdr.de";
+    
+    protected enum UrlType {
+        None,
+        OverviewPage,
+        VideoPage
+    }
+    
+    protected String addDomainIfNecessary(String url) {
+        
+        if(url != null && !url.isEmpty()) {
+            if(url.startsWith("/")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)